test/Libraries/RevitNodesTests/Elements/Views/ViewTests.cs

```diff
@@ -48,6 +45,58 @@ private static View CreateTestView()
 
             return v;
         }
-        
+
+        [Test, TestModel(@".\ViewTemplateTests.rvt")]
+        public void GetViewTemplate()
+        {
+            var rView = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(Autodesk.Revit.DB.View))
+                .FirstOrDefault(x => x.Name == "ViewWithViewTemplate") as Autodesk.Revit.DB.ViewPlan;
+
+            var dView = FloorPlanView.FromExisting(rView, true);
+            Assert.NotNull(dView);
+
+            var viewTemplate = dView.ViewTemplate() as FloorPlanView;
+            Assert.NotNull(viewTemplate);
+        }
+
+        [Test, TestModel(@".\ViewTemplateTests.rvt")]
+        public void SetViewTemplate()
+        {
+            var rView = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(Autodesk.Revit.DB.View))
+                .FirstOrDefault(x => x.Name == "ViewWithoutViewTemplate") as Autodesk.Revit.DB.ViewPlan;
+
+            var dView = FloorPlanView.FromExisting(rView, true);
+            Assert.NotNull(dView);
+
+            var viewTemplate = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(Autodesk.Revit.DB.View))
+                .Cast<Autodesk.Revit.DB.ViewPlan>()
```

Comment:
Sorry for the delay, I just finished running all RevitNodeTests for you, only this one failed when casting,
```
Unable to cast object of Type Autodesk.Revit.DB.View to object of Type Autodesk.Revit.DB.ViewPlan
```